lib/diagnostic.js

```diff
@@ -15,9 +15,18 @@
 
 const fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
```

Comment:
As a rule of thumb, don't delete the boilerplate code that's in the file. You absolutely need these lines.